src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -510,7 +511,9 @@ Context* WriteLog<I>::construct_flush_entry_ctx(
       });
     ctx = new LambdaContext(
       [this, log_entry, read_bl_ptr, ctx](int r) {
-        aio_read_data_block(&log_entry->ram_entry, read_bl_ptr, ctx);
+        auto write_entry = static_pointer_cast<WriteLogEntry>(log_entry);
+        write_entry->inc_bl_refs();
+        aio_read_data_block(write_entry, read_bl_ptr, ctx);
```

Comment:
Let's avoid redundant atomic reference count increments/decrements in new code:
```suggestion
        aio_read_data_block(std::move(write_entry), read_bl_ptr, ctx);
```

Reply:
done